PUBLISHING.rst

```diff
@@ -42,12 +42,8 @@ Once changes have been made to develop that are ready to be published, first cho
 
 Then go through the steps of merging the changes into the master branch:
 
-#. Run :code:`pytest` and make sure all the tests pass. Then run :code:`./test/cmdline_tests.sh` and make sure those tests pass.
 #. Change the 'Unreleased Changes' section of :code:`RELEASE_NOTES.rst` to the new version number.
 #. Check if any changes have been made that have not yet been documented in the release notes. If so, document them.
-#. Update the version number in setup.py
```

Comment:
If not the old way, how are we handling versioning now?

Reply:
the way things are handled now is that when you run `pip install build`, setuptools will automatically create a `version.py` file at the path specified in the pyproject.toml
the version that it uses will be whichever one you used when you created the git tag earlier
I've added a clarification for this that explains the latter part

Reply:
Thank you for explaining!